### PR TITLE
V1.0.0

### PR DESCRIPTION
**Pull request by tortmayr**
- Update changelog
- Update versions for 1.0.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Eclipse GLSP Server Changelog
 
-## v0.1.0 - Upcoming
+## [v0.1.0 - 30/06/2022](https://github.com/eclipse-glsp/glsp-server/releases/tag/v1.0.0)
```
[the remark this answers is not in the document]
Comment (tortmayr) [reply CHANGELOG.md]: :grin: Thanks, good catch!

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,37 +37,37 @@
...
+- [protocol] Added implementation for `SetViewportAction` [#99](https://github.com/eclipse-glsp/glsp-server/pull/99)
+- [model] Introduced `GArgumentable` interface which is implemented by all `GModelElement`s. This can be used to provide additional information in form of an `args` map without having to extend the `GModel` type [#100](https://github.com/eclipse-glsp/glsp-server/pull/100)
+- [model] Extended default type mapping and added builder for `GArgumentable` elements and corresponding utility classes [#105](https://github.com/eclipse-glsp/glsp-server/pull/105)
+- [protocol] Added `fileUri` property to `SaveModelAction` and updated `SaveModelActionHandler` accordingly [#103](https://github.com/eclipse-glsp/glsp-server/pull/103/)
+- [protocol] Added optional `reason` string property to `SetDirtyStateAction`. This property indicates the reason that caused to dirty state change and enables more fine granular handling of dirty state changes [#101](https://github.com/eclipse-glsp/glsp-server/pull/101)
```
Comment (planger) [inline CHANGELOG.md:49]:
```suggestion
- [protocol] Added optional `reason` string property to `SetDirtyStateAction`. This property indicates the reason that caused a dirty state change and enables more fine granular handling of dirty state changes [#101](https://github.com/eclipse-glsp/glsp-server/pull/101)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,37 +37,37 @@
...
+- [di] Complete rework of DI architecture. Move away from one global injector per application. Instead use one global server injector and dedicated child injectors for each diagram session. [#127](https://github.com/eclipse-glsp/glsp-server/pull/127)
+- [launch] Improve customizability of `DefaultGLSPServerLauncher`. [#128](https://github.com/eclipse-glsp/glsp-server/pull/128)
+- [model] Add properties map in model state to store arbitrary properties that are persistent between handler calls. [#132](https://github.com/eclipse-glsp/glsp-server/pull/132)
+- [server] Refactor json-rpc protocol and `DefaultGLSPServerImplementation`. [#133](https://github.com/eclipse-glsp/glsp-server/pull/133)
+- [action] Rework `ActionHandler` & `OperationHandler` API. [#135](https://github.com/eclipse-glsp/glsp-server/pull/135)
+- [action] Introduce `dispatchAfterNextUpdate` functionality for the `ActionDispatcher`. This enables queuing of actions until the next model updates has been processed. [#141](https://github.com/eclipse-glsp/glsp-server/pull/141/)
```
Comment (planger) [inline CHANGELOG.md:58]:
```suggestion
- [action] Introduce `dispatchAfterNextUpdate` functionality for the `ActionDispatcher`. This enables queuing of actions until the next model update has been processed. [#141](https://github.com/eclipse-glsp/glsp-server/pull/141/)
```

**Review by planger — APPROVED** [review]
Thanks, looks good to me! :+1:

**Pull request merged**